### PR TITLE
Add Oracles content

### PR DESCRIPTION
**Pull request by TucksonDev**
Adds 3 pages about Oracles:
- A Concept page: "[Oracles: A conceptual overview](https://nitro-docs-git-add-oracles-section-offchain-labs.vercel.app/for-devs/concepts/oracles)"
- A How to page: "[How to use Oracles in Arbitrum](https://nitro-docs-git-add-oracles-section-offchain-labs.vercel.app/devs-how-tos/how-to-use-oracles)"
- A dev tools and resources page: "[Oracles](https://nitro-docs-git-add-oracles-section-offchain-labs.vercel.app/for-devs/dev-tools-and-resources/oracles)"

### CODE REVIEW

File: arbitrum-docs/devs-how-tos/how-to-use-oracles.mdx [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+description: Learn how to integrate Oracles into your dapp
+reader-audience: developers who want to build on Ethereum/Arbitrum
+content-type: how-to
+---
+
+import PublicPreviewBannerPartial from '../partials/_public-preview-banner-partial.md';
+
+<PublicPreviewBannerPartial />
+
+In this how-to, you’ll learn how to incorporate popular Oracle services into your Arbitrum dApps. Specifically, each section will provide step-by-step instructions on how to obtain current asset prices from oracles, helping you seamlessly integrate these services.
```
Comment (symbolpunk) [inline arbitrum-docs/devs-how-tos/how-to-use-oracles.mdx:13]: @TucksonDev this is one of my favorite docs PRs ever, for so many reasons:

 - Each of the new docs is clearly a specific type of doc
 - Each doc is clearly aligned with structure conventions for its content-type
 - Clear metadata with a description that will catch search queries
 - We set expectations upfront, with an invitation to provide feedback -> clear CTA
 - Headers are sentence-cased, per conventions

All of that to say, the feedback in this review all falls under the category of "nits" / long-tail optimizations. Love love love the thought put into this reader experience 🙏

Comment (symbolpunk) [inline arbitrum-docs/devs-how-tos/how-to-use-oracles.mdx:13]:
```suggestion
Oracles are web services that provide a connection between smart contracts and the outside world. They let decentralized apps (dApps) interact with off-chain data and services.

In this how-to, you’ll learn how to fetch the current price of $ARB tokens from a Solidity smart contract using three different oracles: a **Chainlink** oracle, an **API3** oracle, and a **Tellor** oracle.
```

File: arbitrum-docs/devs-how-tos/how-to-use-oracles.mdx [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+
+This guide assumes that you're familiar with the concept of [Oracles](../for-devs/concepts/oracles.mdx) and have a basic understanding of smart contracts and blockchain development. If you’re new to blockchain development, consider going through our [Quickstart: Build a dApp with Arbitrum (Solidity, Hardhat)](../for-devs/quickstart-solidity-hardhat) before proceeding.
```
Comment (symbolpunk) [inline arbitrum-docs/devs-how-tos/how-to-use-oracles.mdx:15]:
```suggestion
Familiarity with [Oracles](../for-devs/concepts/oracles.mdx), smart contracts, and blockchain development is expected. If you’re new to blockchain development, consider reviewing our [Quickstart: Build a dApp with Arbitrum (Solidity, Hardhat)](../for-devs/quickstart-solidity-hardhat) before proceeding.
```

File: arbitrum-docs/devs-how-tos/how-to-use-oracles.mdx [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+
+## Chainlink
+
+[Chainlink](https://chain.link/) is a widely-recognized Web3 services platform that specializes in decentralized oracle networks. It allows developers to create versatile Web3 applications that can effortlessly connect to real-world data and off-chain computation on Arbitrum and many other blockchain networks. Chainlink offers a variety of data feeds, including asset prices, weather data, random number generation, and many more.
```
Comment (symbolpunk) [inline arbitrum-docs/devs-how-tos/how-to-use-oracles.mdx:19]:
```suggestion
[Chainlink](https://chain.link/) is a widely-recognized Web3 services platform that specializes in decentralized oracle networks. It lets you build Ethereum and Arbitrum dApps that connect to a variety of off-chain data feeds and APIs, including those that provide asset prices, weather data, random number generation, and more.
```

Definitely feel free to discard/partially incorporate if this edit is too "destructive".

**Review by symbolpunk — APPROVED** [review]
I'll continue this review in the AM, but I don't have any blocking feedback and can handle minor nits in a separate PR if you're eager to publish 🤙

File: arbitrum-docs/devs-how-tos/how-to-use-oracles.mdx (hunk 1 above)
Comment (TucksonDev) [reply arbitrum-docs/devs-how-tos/how-to-use-oracles.mdx:13]: Thanks very much! 😃 
I have to say I learned from the best though. Thank _you_ for your constant guidance and support while creating all this content  ❤️

File: arbitrum-docs/devs-how-tos/how-to-use-oracles.mdx [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+        return price;
+    }
+}
+```
+
+You can adapt this contract to your needs. Just remember to use the address of the asset you want to request the price for in the appropriate network, and to **deploy your contract to the same network**. Remember we have a [Quickstart](../for-devs/quickstart-solidity-hardhat) available that goes through the process of compiling and deploying a contract.
+
+### More examples
+
+Refer to [Chainlink’s documentation](https://docs.chain.link/) for more examples for querying price feeds plus other data feeds available.
```
Comment (mahsamoosavi) [inline arbitrum-docs/devs-how-tos/how-to-use-oracles.mdx:76]:
```suggestion
Refer to [Chainlink’s documentation](https://docs.chain.link/) for more examples of querying price feeds plus other data feeds available.
```

File: arbitrum-docs/devs-how-tos/how-to-use-oracles.mdx [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+        // to validate them right after reading from the proxy.
+    }
+}
+```
+
+You can adapt this contract to your needs. Just remember to use the address of the asset you want to request the price for in the appropriate network, and to **deploy your contract to the same network**. Remember we have a [Quickstart](../for-devs/quickstart-solidity-hardhat) available that goes through the process of compiling and deploying a contract.
+
+### Querying a random number through API3
+
+[API3 QRNG](https://docs.api3.org/explore/qrng/) is a public utility provided with the courtesy of [Australian National University (ANU)](https://www.anu.edu.au/). It is served as a public good and is free of charge (apart from the gas costs), and it provides quantum randomness when requiring RNG on-chain.
```
Comment (mahsamoosavi) [inline arbitrum-docs/devs-how-tos/how-to-use-oracles.mdx:130]:
```suggestion
[API3 QRNG](https://docs.api3.org/explore/qrng/) is a public utility provided with the courtesy of [Australian National University (ANU)](https://www.anu.edu.au/). It is served as a public good, it is free of charge (apart from the gas costs), and it provides quantum randomness when requiring RNG on-chain.
```

File: arbitrum-docs/devs-how-tos/how-to-use-oracles.mdx [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+        emit ReceivedUint256(requestId, qrngUint256);
+    }
+}
+```
+
+You can adapt this contract to your needs. Just remember to use the addresses of the appropriate network, and to **deploy your contract to the same network**. Remember we have a [Quickstart](../for-devs/quickstart-solidity-hardhat) available that goes through the process of compiling and deploying a contract.
+
+### More examples
+
+Refer to [API3’s documentation](https://docs.api3.org/) for more examples for querying other data feeds and Airnodes.
```
Comment (mahsamoosavi) [inline arbitrum-docs/devs-how-tos/how-to-use-oracles.mdx:206]:
```suggestion
Refer to [API3’s documentation](https://docs.api3.org/) for more examples of querying other data feeds and Airnodes.
```

**Review by mahsamoosavi — CHANGES_REQUESTED** [review]
A few comments. It's very well-written plus @symbolpunk has said it all :)

File: arbitrum-docs/for-devs/concepts/oracles.mdx
Comment (mahsamoosavi) [inline arbitrum-docs/for-devs/concepts/oracles.mdx:1]: Should we follow our guidelines and use `oracle`?

Comment (TucksonDev) [reply arbitrum-docs/for-devs/concepts/oracles.mdx:1]: Updated: new pages follow the establish guidelines 💪

**Review by mahsamoosavi — APPROVED** [review]

**Pull request merged**